### PR TITLE
Scala 2.13.9 draft release notes

### PR DESCRIPTION
**Pull request by SethTisue**
feedback and suggestions welcome!

### CODE REVIEW

**Comment by SethTisue** [comment]
~TODO: tweak this now that https://github.com/scala/scala/pull/10129 was merged~ done

File: releases/2.13.9.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+* Error on source files with Unicode directional formatting characters ([#10017](https://github.com/scala/scala/pull/10017))
+* Prevent `Function0` execution during `LazyList` deserialization ([#10118](https://github.com/scala/scala/pull/10118))
+
+### Bugfixes
+
+* Emit all bridge methods non-final (perhaps affecting serialization compat) ([#9976](https://github.com/scala/scala/pull/9976))
+* Fix null-pointer regression in `Vector#prependedAll` and `Vector#appendedAll` ([#9983](https://github.com/scala/scala/pull/9983))
+* Improve concurrent behavior of Java `ConcurrentMap` wrapper
+ ([#10027](https://github.com/scala/scala/pull/10027) by [@som-snytt](https://github.com/som-snytt))
```
Comment (som-snytt) [inline releases/2.13.9.md]: please substitute

Igor Gabaydulin <igabaydulin@gmail.com> or https://github.com/igabaydulin

who was the real instigator.

Comment (SethTisue) [reply releases/2.13.9.md]: will do

**Comment by michelou** [comment]
@som-snytt @SethTisue Thank you for ensuring the attractiveness of the 2.13 branch 👏

**Comment by som-snytt** [comment]
For these release notes, it might be necessary to revive the meme with the guy walking with Dotty but whistling over his shoulder at Scala 2.

**Comment by SethTisue** [comment]
> the guy walking with Dotty but whistling over his shoulder at Scala 2

2.13.9's a Som-a-palooza, for sure

**Comment by SethTisue** [comment]
The release notes are now live at https://github.com/scala/scala/releases/tag/v2.13.9

They can be still be edited — feel free to make suggestions here.

**Pull request closed without merging**